src/utility/components.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const Fixed = (Component) => class extends React.Component {
+  shouldComponentUpdate(newProps, newState) {
+    return this.props.style != newProps.style;
+  }
+  render() {
+    return <Component {...this.props} />
+  }
+};
+ 
+export const Pure = (Component) => class extends React.PureComponent {
```

Comment:
There is no difference, it just reads better when using an existing component.

    const PureRow = Pure(Row);